tasty-inspector/src/scala/tasty/inspector/Inspector.scala

```diff
@@ -0,0 +1,25 @@
+package scala.tasty.inspector
+
+import scala.quoted._
+import scala.quoted.runtime.impl.QuotesImpl
+
+import dotty.tools.dotc.Compiler
+import dotty.tools.dotc.Driver
+import dotty.tools.dotc.Run
+import dotty.tools.dotc.core.Contexts.Context
+import dotty.tools.dotc.core.Mode
+import dotty.tools.dotc.core.Phases.Phase
+import dotty.tools.dotc.fromtasty._
+import dotty.tools.dotc.util.ClasspathFromClassloader
+import dotty.tools.dotc.CompilationUnit
+import dotty.tools.unsupported
+import dotty.tools.dotc.report
+
+import java.io.File.pathSeparator
+
+trait Inspector:
+
+  /** Inspect all TASTy files using `Quotes` reflect API */
+  def inspect(using Quotes)(sources: List[String], trees: List[quotes.reflect.Tree]): Unit
```

Comment:
If these are intended to correspond 1:1 (i.e. sources(0) is source of trees(0)), then the documentation should mention so. I think Scala zeitgeist would have us pass a list of pairs, that's more in vogue than doing it the way we do in the compiler and passing two lists.